### PR TITLE
[React] VSCode Extensions

### PR DESCRIPTION
**Pull request by CarlosHWolox**
Added VSCode Extensions import

### CODE REVIEW

File: react/docs/vscode-extensions.md [hunk 1]
```diff
@@ -0,0 +1,45 @@
+# VSCode Extensions
+
+## Introduction
+
+This guide intends to be a guide to useful React/React-Native extensions used along with VSCode. Those extensions are useful to have a productive working enviroment. Also, it will teach how to import/export those extensions.
```
Comment (damfinkel) [inline react/docs/vscode-extensions.md]: `Those extensions are useful for having a productive working environment`

File: react/docs/vscode-extensions.md [hunk 2]
```diff
@@ -0,0 +1,45 @@
...
+
+# VSCode
+
+Citing directly from the page, "Visual Studio Code is a source code editor developed by Microsoft for Windows, Linux and macOS. It includes support for debugging, embedded Git control,syntax highlighting, intelligent code completion, snippets, and code refactoring."
```
Comment (damfinkel) [inline react/docs/vscode-extensions.md]: `Quoting` instead of Citing

File: react/docs/vscode-extensions.md [hunk 3]
```diff
@@ -0,0 +1,45 @@
...
+
+## Import extensions
+
+To import VSCode extensions it is only necessary to use the folowwing command -> code --install + 'extension name'
+
+# Recommended React VSCode Extensions
+
+The following are some recommended extensions to be used, if you have any other extension suggestion you can add it using this form -> https://goo.gl/forms/Oh6pWj3bqgJv2ajp2
+
+## React Extensions
```
Comment (damfinkel) [inline react/docs/vscode-extensions.md]: add Babel JavaScript: https://marketplace.visualstudio.com/items?itemName=mgmcdermott.vscode-language-babel

File: react/docs/vscode-extensions.md (hunk 1 above)
Comment (gon-escalante) [inline react/docs/vscode-extensions.md]: `This guide intends to be a guide` sounds kinda repetitive, i'd change it to something like `This guide will help you install some useful React/React-Native on your VSCode. ...`

**Comment by gussiciliano** [comment]
Hi team, which is the status of this PR?
@CarlosHWolox @damfinkel @WFolini @gon-escalante

**Comment by damfinkel** [comment]
@gussiciliano We are merging this next week or the other at the latest

**Comment by gussiciliano** [comment]
@damfinkel great! :D

**Pull request merged**